python/lsst/ts/imsim/utils/utility.py

```diff
@@ -182,3 +197,106 @@ def __exit__(self, exc_type: None, exc_val: None, exc_tb: None) -> None:
                 os.environ[key] = self._originals[key]
             else:
                 del os.environ[key]
+
+
+def get_cam_type(inst_name: str) -> CamType:
```

Comment:
You don't need this function since `CamType(inst_name)` accomplishes the same thing. It will return the Enum corresponding to the `inst_name`, and if there isn't one defined, it will raise an error.

Reply:
Ok, I was thinking there had to be a way to do this. Thanks.